_data/events.yml

```diff
@@ -1,386 +1,554 @@
   registrations: 42
   participants: ""
-  recording:
-    - title: "7. CNCLinz - Juraci Paixão Kröhling - All you need to know about OpenTelemetry"
-      link: "https://www.youtube.com/watch?v=bh0v6k6eLRw"
+  talks:
+    - title: "All you need to know about OpenTelemetry"
+      speaker: "Juraci Paixão Kröhling"
+      recording: "https://www.youtube.com/watch?v=bh0v6k6eLRw"
 
 - id: 8
-  edition: "8. Cloud Native Computing Linz Meetup"
+  title: "8. Cloud Native Computing Linz Meetup"
   date: "2021-05-25"
   host: "online"
   event_link: "https://www.meetup.com/cloud-native-computing-linz/events/278104169/"
   registrations: 27
   participants: ""
-  recording:
-    - title: "8. CNCLinz - Moritz Wanzenböck - Cloud-Native Storage: What it can and can't (yet) do"
-      link: "https://www.youtube.com/watch?v=DXffA0IgZyc"
+  talks:
+    - title: "Cloud-Native Storage: What it can and can't (yet) do"
+      speaker: "Moritz Wanzenböck"
+      recording: "https://www.youtube.com/watch?v=DXffA0IgZyc"
 
 - id: 9
-  edition: "9. Cloud Native Computing Linz Meetup"
+  title: "9. Cloud Native Computing Linz Meetup"
   date: "2021-06-29"
   host: "online"
   event_link: "https://www.meetup.com/cloud-native-computing-linz/events/278914758/"
   registrations: 15
   participants: ""
-  recording:
+  talks:
     - title: "9. CNCLinz - Charles Pretzer - Service Mesh Principles: Observability through Telemetry with Linkerd"
-      link: "https://www.youtube.com/watch?v=XeXcgz5stMo"
+      speaker: "Charles Pretzer"
+      recording: "https://www.youtube.com/watch?v=XeXcgz5stMo"
 
 - id: 10
-  edition: "10. Cloud Native Computing Linz Meetup"
+  title: "10. Cloud Native Computing Linz Meetup"
   date: "2021-09-28"
   host: "online"
   event_link: "https://www.meetup.com/cloud-native-computing-linz/events/280749046/"
   registrations: 20
   participants: ""
-  recording:
-    - title: "10. CNCLinz - Lars Wolff - Automatically optimize cloud-native systems for fun and profit"
-      link: "https://www.youtube.com/watch?v=0WEts9WLhoc"
+  talks:
+    - title: "Automatically optimize cloud-native systems for fun and profit"
+      speaker: "Lars Wolff"
+      recording: "https://www.youtube.com/watch?v=0WEts9WLhoc"
 
 - id: 11
-  edition: "11. IN PERSON - Cloud Native Computing Linz Meetup"
+  title: "11. IN PERSON - Cloud Native Computing Linz Meetup"
   date: "2021-10-19"
   host: "Dynatrace"
   event_link: "https://www.meetup.com/cloud-native-computing-linz/events/281064988/"
   registrations: 36
   participants: ""
-  recording:
-    - title: "11. CNCLinz - Philipp Weckerle - The 1001 serverless options of Microsoft Azure"
-      link: "https://www.youtube.com/watch?v=81Tg--7GIFI"
+  talks:
+    - title: "The 1001 serverless options of Microsoft Azure"
+      speaker: "Philipp Weckerle"
+      recording: "https://www.youtube.com/watch?v=81Tg--7GIFI"
+    - title: "Cloud-native delivery with Keptn"
+      speaker: "Jürgen Etzlstorfer"
```

Comment:
That actually never happened 😝

Reply:
true that! :) to my defense, I only copied what we had on the meetup.com page 😝 
I have no problem with removing it